repository/src/main/java/com/epam/esm/repository/impl/ClauseBuilder.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.esm.repository.impl;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class ClauseBuilder {
+
+  private static final String EMPTY_STRING = "";
+  private static final String NEW_LINE = "\n";
+
+  private final List<String> parts = new ArrayList<>();
+  private final String start;
+  private final String separator;
+
+  ClauseBuilder(String start, String separator) {
+    this.start = start;
+    this.separator = separator;
+  }
+
+  void addPart(String part) {
+    parts.add(part);
+  }
+
+  String build() {
```

Comment:
I suppose the method can be shortened to:
`return String.join(separator, parts) + NEW_LINE;`

Reply:
fixed